vello_encoding/src/path.rs

```diff
@@ -773,6 +768,39 @@ impl<'a> PathEncoder<'a> {
         };
         Some(pt)
     }
+
+    // Similar to `start_tangent_for_curve` but for a line.
```

Comment:
```suggestion
    /// Similar to [`Self::start_tangent_for_curve`] but for a line.
```
and similarly elsewhere